docs.html

```diff
@@ -14,6 +14,7 @@
   <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
     integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
   <script defer src="./Components/Modal Component/modal.js"></script>
+  <script defer src="./Components/Rating Component/rating.js"></script>
```

Comment:
Suggestion: The js files need to be loaded inside the body at the very end of the files or else it can cause errors as the js files will run first before the HTML gets loaded, thereby not being able to find the elements in the DOM.

Reply:
I've used the defer attribute here , so when DOM parsing encounters defer , it sends network request to fetch the script file ,once it has been fetched it wont process the script until the DOM tree has been constructed, hence there wont be an issue

Reply:
Ohh, I was not aware of that. Thanks for the new info.